example/src/main/java/com/amqtech/opensource/appintroexample/ui/mainTabs/intro/DefaultIntro.java

```diff
@@ -25,20 +25,26 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         sliderPage1.setDescription("This is a demo of the AppIntro library.");
         sliderPage1.setImageDrawable(R.drawable.ic_slide1);
         sliderPage1.setBgColor(Color.TRANSPARENT);
+        sliderPage1.setTitleTypefaceFontRes(R.font.lato);
```

Comment:
As @paolorotolo already mentioned, please move to `DefaultIntro2` or CustomLayout/Background as we want to keep `DefaultIntro` streamlined.